modules/data.atmosphere/DESCRIPTION

```diff
@@ -61,8 +59,8 @@ Suggests:
     progress,
     rlang (>= 0.2.0)
 Remotes:
-    github::ropensci/geonames,
-    github::ropenscilabs/nneo
+    github::ropensci/geonames ,
+    github::ropensci/nneo
```

Comment:
While we're changing this, should update https://github.com/PecanProject/pecan/blob/develop/docker/base/Dockerfile.depends#L22 to match (cc @robkooper)